Tests/Expected/Fonts/swift2-context-defaults-customname.swift

```diff
@@ -20,25 +20,26 @@ struct FontConvertible {
   }
 
   func register() {
-    let bundle = NSBundle(forClass: BundleToken.self)
-
-    guard let url = bundle.URLForResource(path, withExtension: nil) else {
-      return
-    }
-
+    guard let url = url else { return }
     var errorRef: Unmanaged<CFError>?
     CTFontManagerRegisterFontsForURL(url as CFURL, .Process, &errorRef)
   }
+
+  fileprivate var url: NSURL? {
+    let bundle = NSBundle(forClass: BundleToken.self)
+    guard let url = bundle.URLForResource(path, withExtension: nil) else { return nil }
```

Comment:
Wait, given than you `guard let else return nil` and do nothing afterwards, why not directly `return bundle.URLForResource(path, withExtension: nil)` there?